services/device2.lua

```diff
@@ -0,0 +1,3 @@
+--#EVENT {product.id} event
```

Comment:
@ryankucera the dot notation will not work to match the product name.
Maybe just use 'homeproduct' and we document on Readme the product alias must be homeproduct when linking.
Or we can add this to the Config.fallback event with a check on context.service_type == 'P'